CHANGELOG.md

```diff
@@ -29,6 +29,12 @@ The REST API versions in the table are just for your information as the values a
 
 - Visualization rendering stuck due to an error that persisted after the error was gone
 
+## 7.5.0
+
+### Note
+
+- Deprecated version
```

Comment:
We should add some explanation for why is the version deprecated. Something like

> This version was released by accident, please use 7.6.0 instead.

Reply:
Updated.
Thank you